projects/packages/my-jetpack/src/class-rest-zendesk-chat.php

```diff
@@ -13,6 +13,8 @@
  * Registers the REST routes for Zendesk Chat.
  */
 class REST_Zendesk_Chat {
+	const TRANSIENT_EXPIRY   = 1 * MINUTE_IN_SECONDS * 60 * 24 * 7; // 1 week (JWT is actually 2 weeks, but lets be on the safe side)
```

Comment:
Here we can use `WEEK_IN_SECONDS` instead
```suggestion
	const TRANSIENT_EXPIRY   = WEEK_IN_SECONDS; // JWT is actually 2 weeks, but lets be on the safe side
```

Reply:
Thank you!